setup/setup.md

```diff
@@ -1,61 +1,62 @@
+1. Click on the **Add an app** button.
 
-![Sign in to your tenant](images/signintoyourtenant.png)
+1. Enter the name for the app as `Proseware Tasks`, and choose **Create application**.
 
-2. Sign into your tenant (i.e. <user>@<tenant>.onmicrosoft.com, <password>)
+    ![Register your app](images/registeryourapp.png)
 
-![My applications](images/myapplications.png)
+1. The registration page displays, listing the properties of your app.
 
-3. Click on the "Add an app" button.
+1. **IMPORTANT:** Copy the *Application Id* and save it in Notepad. This is the unique identifier for your app. You'll use this **(*) AppId** value to configure your app.
 
-4. Enter the name for the app as "Proseware Tasks", and choose Create application.
+    ![Copy the AppId](images/copytheappid.png)
 
-![Register your app](images/registeryourapp.png)
+1. Under Platforms, choose Add Platform.
 
-5. The registration page displays, listing the properties of your app.
+    ![Add a Platform](images/addaplatform.png)
 
-6. IMPORTANT: Copy the Application Id and save it in Notepad. This is the unique identifier for your app. You'll use this **(*) AppId** value to configure your app.
-	
-![Copy the AppId](images/copytheappid.png)
+1. Choose Web.
 
-7. Under Platforms, choose Add Platform.
-	
-![Add a Platform](images/addaplatform.png)
+    ![Choose Web Platform](images/choosewebplatform.png)
 
-8. Choose Web.
-	
-![Choose Web Platform](images/choosewebplatform.png)
+1. Make sure the Allow Implicit Flow check box is selected, and enter [https://localhost:44382/Home.html](https://localhost:44382/Home.html) as the Redirect URI.
 
-9. Make sure the Allow Implicit Flow check box is selected, and enter https://localhost:44382/Home.html as the Redirect URI.
-10. In Microsoft Graph Permissions, next to Delegated Permissions, click "Add"
-	
-11. Add the permissions so they match the following: 
- 
-![Graph permissions](images/finalgraphpermissions.png)
-	
-12. Choose Save.
+1. In Microsoft Graph Permissions, next to Delegated Permissions, click "Add"
 
-## Get the Planner Task URL from your tenant:
+1. Add the permissions so they match the following screen shot.
 
-1. Use Notepad to open `PlannerSetupInfo.txt` in the root of the repo. Then log into your tenant
+       ![Graph permissions](images/finalgraphpermissions.png)
 
-![Log into your tenant](images/tenantlogin.png)
+1. Choose Save.
 
-2. Open the Planner app:
+## Get the Planner Task URL from your tenant
 
-![Open Planner](images/openplanner.png)
+1. Use Notepad to open `PlannerSetupInfo.txt` in the `/setup` folder of the repo. Then sign in to your tenant.
 
-3. In the address bar of the browser, copy the full URL. This will be the **(2) tenant-specific base URL** for use later in the code:
+    ![Log into your tenant](images/tenantlogin.png)
 
-## Update the code for your tenant:
+1. Open the **Planner** app.
+
+    ![Open Planner](images/openplanner.png)
+
```

Comment:
Should we add some information here about skipping the welcome dialog and cancelling creating a new plan?

Reply:
Sure that would be good. Please add it.